pkg/controlplaneclient/runner.go

```diff
@@ -142,12 +142,14 @@ func (c *client) WatchRunnerRequests(ctx context.Context) RunnerRequestsWatcher
 	}
 	go func() {
 		defer func() {
+			c.logger.Errorw("shutting down watch runner requests", "error", err)
 			cancel(err)
 			watcher.Close(err)
 		}()
 		for {
 			// Ignore if it's not implemented in the Control Plane
 			if getGrpcErrorCode(err) == codes.Unimplemented {
+				c.logger.Errorw("watch runner requests", "error", err)
```

Comment:
I'd rename it to `error watching runner requests` so it is more explicit (same for below)